src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -60,8 +60,10 @@ def get_sysctl(self, var):
 
     @private
     def set_sysctl(self, var, value):
-        with open(f'/proc/sys/{var.replace(".", "/")}', 'w') as f:
-            f.write(value)
+        path = f'/proc/sys/{var.replace(".", "/")}'
```

Comment:
Do you think we should perhaps have validation to now allow such tunables in the first place ?